components/jp_funda/package.json

```
@@ -1,19 +1,18 @@
 {
   "name": "@pipedream/jp_funda",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

[New `minor` version](https://pipedream.com/docs/components/guidelines/#versioning) since there are features being added
```suggestion
  "version": "0.1.0",
```